library/psa_crypto.c

```diff
@@ -1179,6 +1179,9 @@ static psa_algorithm_t psa_key_policy_algorithm_intersection(
 static int psa_key_algorithm_permits( psa_algorithm_t policy_alg,
```

Comment:
The algorithm policy hierarchy is also reflected in `psa_key_policy_algorithm_intersection`, so that function needs to understand the new policies as well.

Reply:
Fixed.